<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADFGVX Cipher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>plaintext</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plaintext_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>keysquare = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="square_edit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>keyword = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyword_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="encrypt_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>v Encrypt v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypt_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>^ Decrypt ^</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ciphertext</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ciphertext_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="random_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>random keysquare</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
